packages/ddp-server/crossbar.js

```diff
@@ -6,12 +6,21 @@ DDPServer._Crossbar = function (options) {
   var self = this;
   options = options || {};
 
-  self.nextId = 1;
+  self.listenerId = 1;
+  self.bufferId = 1;
   // map from collection name (string) -> listener id -> object. each object has
   // keys 'trigger', 'callback'.  As a hack, the empty string means "no
   // collection".
   self.listenersByCollection = {};
   self.listenersByCollectionCount = {};
+  // An object which holds the buffered changes per collection
+  self.buffersPerCollection = {};
+  // Buffer changes to the same collection which happen within x ms
+  self.bufferInterval = 5;
+  // Maximum age of the buffer
+  self.bufferMaxAge = 500;
+  // Maximum amount of notifications to store in the buffer before flushing
+  self.bufferMaxSize = 1000;
```

Comment:
These values (5, 500, 1000) are somewhat arbitrary, I guess mileage may vary.

Reply:
Might be worth making these configurable.